homeassistant/components/unifi/controller.py

```diff
@@ -61,13 +60,14 @@
     DEFAULT_TRACK_DEVICES,
     DEFAULT_TRACK_WIRED_CLIENTS,
     DOMAIN as UNIFI_DOMAIN,
+    HEATBEAT_MISSED_SIGNAL,
```

Comment:
Could we define the signal as a property inside the controller like the other signals?

Reply:
I may not be following your comment fully, but.... since its the unique_id of the entity that is missing the heartbeat I don't think we can make it a property since its going to be different for every watched device:

```
                async_dispatcher_send(
                    self.hass, f"{HEARTBEAT_MISSED_SIGNAL}_{unique_id}"
                )
```
Let me know if I missed something.

Reply:
I mostly want to replace the HEARTBEAT_MISSED_SIGNAL part to keep it with how other signals are based

```
 async_dispatcher_send(self.hass, f"{self.controller.signal_heartbeat_missed}_{unique_id}")
```

Reply:
done

Reply:
Hopefully I did what you were expecting. 👍  My turn for it to be after midnight, so I'll give it another look in the morning + retest when I've had some more sleep. G'night!

Reply:
Gnite!

Yes it looks fine for me. 🎉